services/static-webserver/client/source/class/osparc/data/model/NodeProgressSequence.js

```diff
@@ -206,29 +200,35 @@ qx.Class.define("osparc.data.model.NodeProgressSequence", {
       const clusterUpScalingPBar = this.__clusterUpScalingPBar = this.self().createProgressBar();
       this.__sequenceLoadingPage.add(clusterUpScalingPBar);
 
-      const pullingInputsTitle = this.__pullingInputsTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Pulling inputs..."));
-      this.__sequenceLoadingPage.add(pullingInputsTitle);
+      const pullingSidecarTitle = this.__pullingSidecarTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Setting up key components..."));
+      this.__sequenceLoadingPage.add(pullingSidecarTitle);
 
-      const pullingInputsPBar = this.__pullingInputsPBar = this.self().createProgressBar();
-      this.__sequenceLoadingPage.add(pullingInputsPBar);
+      const pullingSidecarPBar = this.__pullingSidecarPBar = this.self().createProgressBar();
+      this.__sequenceLoadingPage.add(pullingSidecarPBar);
 
-      const pullingOutputsTitle = this.__pullingOutputsTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Pulling outputs..."));
+      const pullingOutputsTitle = this.__pullingOutputsTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Downloading outputs..."));
       this.__sequenceLoadingPage.add(pullingOutputsTitle);
 
       const pullingOutputsPBar = this.__pullingOutputsPBar = this.self().createProgressBar();
       this.__sequenceLoadingPage.add(pullingOutputsPBar);
 
-      const pullingStateTitle = this.__pullingStateTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Pulling state..."));
+      const pullingStateTitle = this.__pullingStateTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Retrieving your work..."));
```

Comment:
Downloading your files...

Reply:
Q1: why make it more technical?
Q2: inputs and outputs also contain files. I think it becomes a bit inconsistent.

Reply:
Mmh, true, but also outputs are part of "your work".

What about "Downloading your workspace"? Once they have used it one/two types, they should know what the "workspace" folder is.

If you don't like it, "Retrieving your work" also works.

Reply:
While outputs are part of your work. The term work is a more generic concept, unlike workspace which is a bit more technical. Not all services will call the state folder as worksapce

Reply:
Personally I wouldn't use downloading as it sounds like you are downloading them to your computer not the workspace. Perhaps something like:

Retrieving assets... (Then include inputs and outputs in this why have extra details)

Then something like 

Generating workspace...